ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -64,6 +64,11 @@ tests:
   commands: make ci-job
```

Comment:
Should we add here `skip_if_only_changed: ^ztp` then?

Reply:
no. because it has stuff that related to ztp also. i.e.: "verify-commits gofmt golint govet cnftests-unit"

Reply:
explaining diffidently - the ztp has additional checks on top of the basic